stable/datacube-ows/values.yaml

```diff
@@ -39,12 +50,14 @@ ows:
   livenessProbe: {}
   readinessProbe: {}
   startupProbe: {}
+  securityContext: {}
```

Comment:
@woodcockr - Could you please explain why we have security context twice and what the difference between the 2 of them is ?

Reply:
No idea, wasn't added by me and I didn't notice it ;-)


There are two Containers - which would be why there are two - if owsConfig uses an image for configuration.

@mmochan - What's the deal with the securityContext?

Reply:
securityContext is useful specify security settings for a Pod. Sometime pod running as non root user not able to read the AWS web identity token file. Then configuring `securityContext` can help. Not sure is that the intention here.